.circleci/phpcs.sh

```diff
@@ -16,5 +16,5 @@ composer global require drupal/coder
 composer global require dealerdirect/phpcodesniffer-composer-installer
 
 for file in "${targets[@]}"; do
-  [ -f "${file}" ] && phpcs --standard=Drupal,DrupalPractice -s --colors "${file}"
+  [ -f "${file}" ] && /home/.composer/vendor/bin/phpcs --standard=Drupal,DrupalPractice -s --colors "${file}"
```

Comment:
We may need to get this in as a standalone commit to unblock current CI failures